### ISSUE TITLE
Further tests with the current setup

### ISSUE THREAD

**Issue by AndreWeiner** [AndreWeiner/wall_modeling#6]
Hi @JihooKang-KOR,

I had some thoughts about further factors that could potentially influence the resulting friction coefficient. Therefore, I suggest the following tests:
- test different schemes for `div(phi,U)` and the SA turbulence model (as discussed in the meeting)
- take the mesh with the lowest y-plus value, double the number of cells along the plate (x-direction), and see if the results change (for both turbulence models, no wall functions, tolerance of 10^-5 for p and U)
- plot c_f curves comparatively for different y-plus values (currently, they are not directly compared); I suggest using the curves obtained with the tolerance 10^-5; compare these curves between the two turbulence models

Best, Andre

**Comment by AndreWeiner** [comment]
Dear Jihoo,

thanks for the extensive visualizations. I have noticed the following after a first look at the data:
- the jumps in the Cf curves occur only for **yPlus=10**; that behavior is largely  independent of the mesh resolution, the coupling tolerance, the turbulence model, the use of wall functions, or the divergence scheme; divergence scheme and turbulence model change the *wigglyness* a bit, but the jumps are present in all the curves
- for yPlus=10 I noticed that the residuals stop decreasing at a quite high tolerance compared to the other cases

When looking at the Ux, Uy, p fields for yPlus=10, do you notice anything strange compared to other cases?

Best, Andre

**Comment by JihooKang-KOR** [comment]
Dear Andre,

If I understand correctly, Ux, Uy, and p fields (not residual) themselves should be plotted. Then, I need a few days to simulate again to extract Ux, Uy, and p from each final time folder and plot them.

Therefore, please wait for a few days to get the results if my understanding is correct.

Best regards,
Jihoo Kang

**Comment by AndreWeiner** [comment]
Sure, no problem.
Best, Andre

**Comment by JihooKang-KOR** [comment]
Dear Andre,

The jumps at y+ = 10 happen in three cases (SA model with/without wall functions, and kOmegaSST model without wall functions), and therefore I checked those cases for y+ = 1, 10, and 30.

1. **U fields** : As seen in the figure, there isn't anything special distribution of velocity visibly (looks like a natural behavior). Here, I uploaded only the case of '*SpalartAllmaras*' model, but the comprehensible status is almost the same. For '*SpalartAllmaras*' model without wall functions, the fields at the back part of the plate were checked because the jumps only happen there in this case. Otherwise, the front part was checked.

![image](https://user-images.githubusercontent.com/75588742/119464066-afcef800-bd42-11eb-9a6f-16374d1a401e.png)
<'*SpalartAllmaras*' model at y+ = 1 without Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119464160-c2e1c800-bd42-11eb-8615-f7ed58bff383.png)
<'*SpalartAllmaras*' model at y+ = 10 without Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119464261-d725c500-bd42-11eb-8ac3-5697e0e39f92.png)
<'*SpalartAllmaras*' model at y+ = 30 without Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119464780-6206bf80-bd43-11eb-8118-8489c1eac74f.png)
<'*SpalartAllmaras*' model at y+ = 1 with Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119464830-6fbc4500-bd43-11eb-808d-616c252a146b.png)
<'*SpalartAllmaras*' model at y+ = 10 with Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119464903-7e0a6100-bd43-11eb-80ed-779006b2b386.png)
<'*SpalartAllmaras*' model at y+ = 30 with Wall Functions>

2. **p fields** : It seems to be more difficult to distinguish anything in the p fields for the same region with the U fields. Therefore, this fields do not yield anything meaningful as well.

![image](https://user-images.githubusercontent.com/75588742/119465283-d5103600-bd43-11eb-810e-eec1901a8ad4.png)
<'*kOmegaSST*' model at y+ = 1 without Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119465402-eb1df680-bd43-11eb-983c-d9f540bcbecc.png)
<'*kOmegaSST*' model at y+ = 10 without Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119465467-fa04a900-bd43-11eb-99bb-2b5af0b2a6d5.png)
<'*kOmegaSST*' model at y+ = 30 without Wall Functions>

3. **Kink in '*kOmegaSST*' model** : Meanwhile, in '*kOmegaSST*' model, there is a kink at the front of the plate except few cases. I checked p fields at y+ = 10 with/without wall functions, since there is no kink with wall functions, whereas a kink emerges without wall functions at y+ = 10. As seen below, I discovered that there is a significant pressure drop at the front part right after the plate starts. We are not able to figure out the reason why it happens, but at least we can conclude that the kink is observed in pressure fields, while velocity fields do not give any related information.

![image](https://user-images.githubusercontent.com/75588742/119466762-3b498880-bd45-11eb-82e5-0fd250c32492.png)
<'*kOmegaSST*' model at y+ = 10 without Wall Functions>

![image](https://user-images.githubusercontent.com/75588742/119466854-4dc3c200-bd45-11eb-8f1d-ee13d0f0c903.png)
<'*kOmegaSST*' model at y+ = 10 with Wall Functions>

In conclusion, I could not find any strange behavior for p and U fields regarding the jumps.

Best regards,
Jihoo Kang
